src/runtime/security/meta_access_controller.cpp

```diff
@@ -65,14 +65,10 @@ meta_access_controller::meta_access_controller(
                                         "RPC_CM_NOTIFY_STOP_SPLIT",
                                         "RPC_CM_QUERY_CHILD_STATE",
                                         "RPC_NEGOTIATION",
-                                        "RPC_CALL_RAW_MESSAGE",
```

Comment:
Why remove these rpc codes?

Reply:
coding error，duplicated in  input-parameter

Reply:
OK, let's adjust the rpc codes in lexicographical order to avoid these mistakes?